duckduckgo/iOS
# pp tracker network screen

Reviewer: Mia
Asana:

* https://app.asana.com/0/414235014887631/457416080701395 
* https://app.asana.com/0/414235014887631/457416080701396

CC:

**Description**:

Tracker network screen shows the networks, domains and categories of trackers for all networks.

Also implemented major networks screen which is the same but filtered by major networks.

**Steps to test this PR**:

1. Visit cnn.com
1. Open privacy protection
1. Observe large number of networks blocked (around 11) with trackers, categories and icons 
1. Tap "tracker networks blocked"
1. Observe list
1. Go back and tap "major tracker networks found"
1. Observe same list but only for major networks with appropriate title
1. Visit encryption and network leaderboard screens
1. Observe the icons at the top of the screens are red

1. Visit DDG 
1. Repeat above test and observe all icons are green and the networks found screen is empty
